src/vectorizationbase_compat/contract_pass.jl

```diff
@@ -107,20 +107,20 @@ function recursive_muladd_search!(call, argv, cnmul::Bool = false, csub::Bool =
                         csub = false
                     end
                     return true, cnmul, csub
-                end                
+                end
             end
         end
     end
     length(call.args) == 4, cnmul, csub
 end
-                          
-function capture_muladd(ex::Expr, mod, LHS = nothing)
+
+function capture_muladd(ex::Expr, mod)
     call = Expr(:call, Symbol(""), Symbol(""), Symbol(""))
     found, nmul, sub = recursive_muladd_search!(call, ex.args)
     found || return ex
     # a, b, c = call.args[2], call.args[3], call.args[4]
     # call.args[2], call.args[3], call.args[4] = c, a, b
-    clobber = false#call.args[4] == LHS
+    clobber = false
     f = if nmul && sub
         clobber ? :vfnmsub231 : :vfnmsub
```

Comment:
Could remove `clobber` as well, since it's trivially `false`

Reply:
Good point, I didn't really look very carefully.